### DISCUSSION TITLE
adjust the sentence length and the way how the model will split sentences

### DISCUSSION THREAD

**Discussion by AIHIsora** [absadiki/subsai#19] [Q&A]
Thanks for the great work! It's the most accurate tools to transcribe audios as far as I known.

But in my test I met with a problem that the model will split sentences in some unreasonable ways. Sometimes the clips are split in the middle of a sentence, and sometimes one clip contain a long blank between two sentences. Is there a parameter to adjust it to get better segment performance? 
I'm now testing it with Japanese, if it matters.

**Comment by absadiki** [comment]
Hi @AIHIsora,

You are welcome. 'Glad you found it useful. 

Could you please elaborate more ? what model are you using ? have you tried other media files (other than Japanese) ?
Have you tried the provided VAD example:
> * [VAD example](https://github.com/abdeladim-s/subsai/blob/main/examples/subsai_vad.ipynb): process long audio files using [silero-vad](https://github.com/snakers4/silero-vad). <a target="_blank" href="https://colab.research.google.com/github/abdeladim-s/subsai/blob/main/examples/subsai_vad.ipynb">
  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
</a>

In general, you can get better performance by using a bigger model size!

**Reply by AIHIsora** [reply]
I was using whisper medium model (limited by my gpu memory), and only test in Japanese. I found the problem may be due to my test media, which is a mix of songs and talks that will contaminate each other. The VAD example seems to be a suitable solution, or maybe I can split the song and talk chunks manually. 
Thank you for the reply. I will test further and will come back with feedback if I make progress or encounter new problems.
